### ISSUE TITLE
Breadcrumbs doesn't show when folder name is same as nginx root folder.

### ISSUE THREAD

**Issue by flatblowfish** [DirectoryLister/DirectoryLister#671]
**Directory Lister Version** is v3.8.3.
**PHP Version** is 8.

**Describe the bug**

I'm looking for a simple lister program, I found DirectoryLister. So I test it, make sure it suits my needs. Here is my problem.

I have a domain name, for my privacy, just call it `domain.com`.
I want to visit `download.domain.com`, to list files under `/usr/share/nginx/html/download`.

My nginx config:

```conf
location / {
    root   /usr/share/nginx/html/download;
    index  index.html index.php;
}

location ~ \.php$ {
    root           /usr/share/nginx/html/download;
    fastcgi_pass   php8:9000;
    fastcgi_index  index.php;
    fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
    include        fastcgi_params;
}
```

**I found that if mkdir `/usr/share/nginx/html/download/download`, then breadcrumbs in web page, did not show up.** If mkdir `/usr/share/nginx/html/download/download/download`, same situation. Maybe I doing something wrong, or it is a bug.

![1](https://user-images.githubusercontent.com/87540244/128611067-9367e63e-0ff1-4d8d-a91d-29e1127f78b0.png)

![2](https://user-images.githubusercontent.com/87540244/128611075-a11b126d-415a-4ed3-bdd0-d171c79a0c88.png)

**Comment by PHLAK** [comment]
Interesting. Thanks for reporting this, I will look into it.

**Comment by PHLAK** [comment]
I was able to reproduce this and in fact if you had a directory with the name of ANY of the folders in your installation's base path they would be stripped. For example, if you installed Directory Lister to the folder `/var/www/downloads` then any path containing a folder name of `var`, `www` or `downloads` would be missing the breadcrumb for that folder.

Resolved with 09a78e2

**Comment by flatblowfish** [comment]
Thanks! My first github issue. 
See all the possibilities in programming is impossible, most of the time just think it is so. I think this cause most of bugs.
